### PR TITLE
Feature tests for signing out and logging into Rails admin

### PR DESCRIPTION
**Pull request by pitchinvasion**

### CODE REVIEW

File: app/views/layouts/_messages.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- flash.each do |name, msg|
+  %div{:class => "alert-box round #{name.to_s == 'notice' ? 'success' : 'alert'}", "data-alert" => ""}
```
Comment (silvabox) [inline app/views/layouts/_messages.html.haml]: I find this style much harder to read than haml's other formatting options e.g. the following line produces the same result:

`.alert_box.round(data-alert){class: name.to_s == 'notice' ? 'success' : 'alert'}

File: app/views/layouts/_messages.html.haml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    = msg
+    %a.close{:href => "#"} ×
```
Comment (silvabox) [inline app/views/layouts/_messages.html.haml]: similarly, I think `%a.close(href="#")` is easier to read.

Comment (spike01) [inline app/views/layouts/_messages.html.haml]: `%a.close(href: "#")` no?

File: spec/features/rails_admin_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+RSpec.feature "Rails admin login" do
+
+  let(:admin) { FactoryGirl.create(:admin) }
+
+  context "As a logged in admin" do
+    before do
+      login_as(admin, scope: :admin)
+    end
+
+    scenario "I access the Rails admin page" do
```
Comment (spike01) [inline spec/features/rails_admin_spec.rb]: "I can access" cuz, consistency

File: spec/features/rails_admin_spec.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      visit rails_admin_path
+      expect(page).to have_content(admin.email)
+    end
+  end
+
+  context "As an anonymous user" do
+    scenario "I cannot access Rails admin" do
```
Comment (spike01) [inline spec/features/rails_admin_spec.rb]: "the Rails admin"

**Pull request merged**